app/Jobs/ForgetPasswordMail.php

```diff
@@ -27,7 +27,7 @@ public function __construct(private $user, private $otp)
      */
     public function handle(): void
     {
-        $data = ['otp' => $this->otp, 'name' => $this->user->name];
+        $data = ['otp' => $this->otp, 'firstname' => $this->user->firstname, 'lastname' => $this->user->lastname];
```

Comment:
Instead of use **$this->user** 3 times we can pass it from the __construct method